CaveRunner/Assets/CaveRun3D/Scripts/GameController.js

```diff
@@ -56,7 +59,16 @@ function Update ()
  		// Ignore back button
  	}
 
+ 	if (Paused) {
+ 		Time.timeScale = 0;
+ 		return;
+ 	} else {
+ 		Time.timeScale = 1.0f;
```

Comment:
huh i'm wonder this is highlighted

Reply:
I was wondering this, too, but it didn't seem to affect anything